tests/integration/index/create_unique_test.go

```diff
@@ -295,8 +293,6 @@ func TestUniqueIndexCreate_UponAddingDocWithExistingNilValue_ReturnError(t *test
 					{
 						"name":	"Andy"
 					}`,
-				ExpectedError: db.NewErrCanNotIndexNonUniqueFields(
-					"bae-2159860f-3cd1-59de-9440-71331e77cbb8", errors.NewKV("age", nil)).Error(),
 			},
 		},
 	}
```

Comment:
added a test action `GetIndexes` that is run after `CreateIndex` while having exinsting docs with multiple `nil` values.
For the rest of the test it pointless because they create index along with the collection and there are already tests for this.
Note: creating docs can not delete index.